stack-ghc-9.2.yaml

```diff
@@ -0,0 +1,41 @@
+resolver: nightly-2021-02-01
+compiler: ghc-9.2.1
+
+packages:
+- servant-client-core/
+- servant-client/
+- servant-docs/
+- servant-foreign/
+- servant-http-streams/
+- servant-server/
+- servant/
+
+- servant-conduit
+- servant-machines/
+- servant-pipes/
+- servant-swagger/
+
+extra-deps:
+# https://github.com/GetShopTV/swagger2/pull/231
+- github: GetShopTV/swagger2
+  commit: 1313547f8ebe594198a1486f0592a962da83e8c6
+
+# https://github.com/haskell-foundation/foundation/pull/555
+- github: haskell-foundation/foundation
+  commit: 0bb195e1fea06d144dafc5af9a0ff79af0a5f4a0
+  subdirs:
+    - basement
+
+# https://github.com/vincenthz/hs-memory/pull/87
+- github: vincenthz/hs-memory
+  commit: 3cf661a8a9a8ac028df77daa88e8d65c55a3347a
+
+# https://github.com/haskell-crypto/cryptonite/pull/354
+- github: haskell-crypto/cryptonite
+  commit: 3b081e3ad027b0550fc87f171dffecbb20dedafe
+
+# https://github.com/aesiniath/http-streams/pull/126
+- github: aesiniath/http-streams
+  commit: 86cd7f970d747ee653328e59176d451780641992
```

Comment:
These lines are now unnecessary since http-streams 0.8.9.6 have been released containing aesiniath/http-streams/pull/126 (which you submitted)